build-scripts/gulp/translations.js

```diff
@@ -40,20 +40,17 @@ class CustomJSON extends Transform {
     this._reviver = reviver;
   }
 
+  // eslint-disable-next-line @typescript-eslint/naming-convention
   async _transform(file, _, callback) {
-    try {
```

Comment:
Why are the try/catch blocks removed? Do the errors now surface? Or...?

Reply:
Gulp is handling the error and shows it to you. But when we catch it and do a resolve callback it won't show it and fails on the wrong part and can't tell you what actually failed.